core/lib/tasks/exchanges.rake

```diff
@@ -20,12 +20,13 @@ namespace :exchanges do
       begin
         item_charger.charge_for_items
       rescue Spree::UnreturnedItemCharger::ChargeFailure => e
-        failure = {message: e.message, new_order: e.new_order.try(:number)}
-      rescue Exception => e
-        failure = {message: "#{e.class}: #{e.message}"}
+        failure = { message: e.message }
+      rescue => e
```

Comment:
I think I'd just collapse these and not think about rescuing non-`StandardError` exceptions. Like, what do you even do with a `NoMemoryError`? But I don't feel strongly about that. Doesn't seem clear to me what else to do though.